pkg/config/config.go

```diff
@@ -64,14 +66,26 @@ type Configuration struct {
 	Dapr *DaprConf `json:"dapr" yaml:"dapr"`
 	// Log log configuration.
 	Log *LogConf `json:"log" yaml:"log"`
+	// AuthMysql  mysql config of auth.
+	AuthMysql *authConf.MysqlConf `json:"auth_mysql" yaml:"authMysql"`
+	// RBAC rbac config of auth.
+	RBAC *authConf.RBACConfig `json:"rbac" yaml:"rbac"`
+	// OAuth2Config oauth2 config of auth.
+	OAuth2Config *authConf.OAuth2Config `json:"oauth2_config" yaml:"oauth2"` // nolint
+	// entity entity security config of auth.
+	Entity *authConf.EntityConfig `json:"entity" yaml:"entity"`
 }
```

Comment:
```suggestion
        // SecurityConf security config.
        Security *SecurityConf `json:"security" yaml:"security"`
}

type SecurityConf struct {
	// Mysql  mysql config of security.
	Mysql *security_conf .MysqlConf `json:"mysql" yaml:"mysql"`
	// RBAC rbac config of security.
	RBAC *security_conf .RBACConfig `json:"rbac" yaml:"rbac"`
	// OAuth2Config oauth2 config of security.
	OAuth2 *security_conf .OAuth2Config `json:"oauth2" yaml:"oauth2"` // nolint
	// entity entity security config of auth.
	Entity *security_conf .EntityConfig `json:"entity" yaml:"entity"`
}
```
把这几个 security 的配置放在一个结构里面给加进去可以不？